src/components/Books.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Navbar from './Navbar';
+
+const Books = () => {
+  const booksArray = [{
+    id: 1, title: 'Dune', author: 'Frank Herbert', genre: 'Science Fiction',
+  }];
+  return (
+    <>
+      <Navbar />
+      <h1 className="books-page-title">E-Bookopolis</h1>
+      <form id="form">
+        <input type="text" id="book-title" name="book-title" className="book" placeholder="Book Name..." />
+        <select name="categories" id="categories-dropdown">
+          <option value="" disabled selected hidden>Categories</option>
+          <option value="horror">Horror</option>
+          <option value="sci-fic">Sci-Fic</option>
+          <option value="comedy">Comedy</option>
+        </select>
+        <input type="submit" value="Submit" />
+      </form>
+      <ul className="books-container">
+        {
+              booksArray.map((book) => (
+                <li key={book.id}>
+                  <ul>
+                    <li>{book.genre}</li>
+                    <li>{book.title}</li>
+                    <li>{book.author}</li>
+                  </ul>
+                  <input type="button" className="remove-btn" id={book.id} value="Remove" />
```

Comment:
- [ ] There should be a component responsible for a single book. Kindly extract these highlighted lines into a new component (eg Book/SingleBook/whatever name you wish). Then render that book here instead with book data as a prop (see example image).

  ![hammas](https://user-images.githubusercontent.com/36542986/154993308-72d8f06c-9ba3-4846-b2bc-246f750578e4.jpg)

Reply:
I have implemented this change.